test/integration_test/common_test.go

```diff
@@ -440,16 +456,50 @@ func verifyScheduledNode(t *testing.T, appNode node.Node, volumes []string) {
 			}
 		}
 	}
+	return scores
+}
 
-	highScore := 0
-	for _, score := range scores {
-		if score > highScore {
-			highScore = score
-		}
+// Verify nodes are scheduled on appropriate nodes based on hyperconvergence scoring
+// This method assumes we have pod AntiAffinities set in the deployment to deploy each replica on a different node.
+func verifyScheduledNodesMultipleReplicas(t *testing.T, appNodes []node.Node, volumes []string) {
+	driverNodes, err := storkVolumeDriver.GetNodes()
```

Comment:
Let's check with Rohit about how to get the testRailID and make changes similar to following 
```	
var testrailID, testResult = 84664, testResultFail
	runID := testrailSetupForTest(testrailID, &testResult)
	defer updateTestRail(&testResult, testrailID, runID)
```
@Rohit-PX Can you provide a brief description of how do we get the testRailID?  
Do we need to manually create a test in testrail first and it automatically generates the id that can be used here?

Reply:
@olavangad-px @pp511 we will have to add the test to testrail first (if it does not exist) and get it from testrail.

Reply:
Have added the test in testrail https://portworx.testrail.net/index.php?/cases/view/92964&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=3252 and updated the code in the latest commit